sdk/attestation/azure-security-attestation/CHANGELOG.md

```diff
@@ -1,12 +1,6 @@
 # Release History
 
-## 1.0.0.beta.1 (Unreleased)
+## 1.0.0-beta.1 (2022-04-05)
 
 ### Features Added
 * Attestation Package creation
```

Comment:
We use `-` instead of `*`
And there should be one empty line between a tittle and the bullets
```suggestion

- Attestation Package creation
```

Reply:
Should I also make this change in template, since that's where I copied this from?